functions/pipeline/onboarding/__init__.py

```diff
@@ -1,6 +1,23 @@
+import os
 import logging
 import azure.functions as func
+from ..shared import db_access_v2 as DB_Access_V2
+from azure.storage.blob import BlockBlobService, ContentSettings
 
+default_db_host = ""
+default_db_name = ""
+default_db_user = ""
+default_db_pass = ""
+
+# TODO: Make environment variables
+storage_account_name = ""
+storage_account_key = ""
+source_container_name = ""
+destination_container_name = ""
+
+# TODO: Make environment variables
+SOURCE_CONTAINER_URL = "https://akaonboardingstorage.blob.core.windows.net/aka-temp-source-container"

```

Comment:
Probably want these two to be env vars as well, no need to check this into the repository

Reply:
Fixed.